src/go.mod

```diff
@@ -83,3 +87,5 @@ require (
 	k8s.io/client-go v0.17.3
 	k8s.io/helm v2.16.3+incompatible
 )
+
+replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.3+incompatible
```

Comment:
why we need the `replace` here?

Reply:
to resolve the azure go-autotest versioning issue, similar things in k8s world:

https://github.com/kubernetes-sigs/external-dns/blob/master/go.mod#L68
https://github.com/kubernetes/kubernetes/blob/master/go.mod#L155

We can group the replace, but so far it's only one here.